content/sensu-core/0.29/reference/filters.md

```diff
@@ -297,6 +297,131 @@ tokens"` will be published to the Sensu server log._
 
 ## Filter configuration
 
+### Built-in Filters - occurrences {#built-in-filters-occurrences}
+
+The `occurrences` filter is included in every install of Sensu. This filter can 
+be applied to a handler using the "filter" or "filters" handler definition attribute.
+
+For example:
+{{< highlight json >}}
+{
+  "handlers": {
+    "email": {
+      "...": "...",
+      "filters": ["occurrences"]
+    }
+  }
+}
+{{< /highlight >}}
+
+
+The `occurrences` filter uses two custom check definition attributes, `occurrences` 
+and `refresh`. 
+
+`occurrences`: The number of events that must occur before an event is handled for 
+the check (default is 1).
+
+`refresh`: Time in seconds until event occurrences are handled for the check again 
+(default is 1800). For example, a check with a refresh of 1800 will have its' event
+(recurrences) handled every 30 minutes, to remind users of the issue.
+
+For example:
+{{< highlight json >}}
+{
+  "checks": {
+    "check-http": {
+      "...": "...",
+      "occurrences": 2,
+      "refresh": 3600
+    }
+  }
+}
+{{< /highlight >}}
+
+### Built-in Filters - check_dependencies {#built-in-filters-check-dependencies}
+
+The `check_dependencies` filter is included in every install of Sensu. This 
+filter can be applied to a handler using the "filter" or "filters" handler definition
```

Comment:
I'm leery of putting "filter" in here. Best practice would be for customers to use "filters".